EMML/7001-8000/EMML7638.xml

```diff
@@ -35,19 +36,233 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
+
+<msItem xml:id="ms_i1.14">
+<locus from="84r" to="89v"/>
+<title type="complete" ref="LIT6441Kehda"/>
+<note>Indication for reading: <foreign xml:lang="gez">ዐርብ፡ ነግህ፡ ምንባብ፡</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.15">
+<locus from="90r" to="96r"/>
+<title type="complete" ref="LIT6444HomilyPilatos"/>
+<note>Indication for reading: <foreign xml:lang="gez">በዐርብ፡ በ፱፡ ሰዐት፡ ምንባብ፡</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="96v" to="108r"/>
+<title type="complete" ref="LIT5862OnSpiritualEaster"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="108r" to="113v"/>
+<title type="complete" ref="LIT2109OntheA"/>
+<note>Indication for reading: <foreign xml:lang="gez">በዓለ፡ ፵፡ ምንባብ፡</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.18">
+<locus from="114r" to="127r"/>
+<title type="complete" ref="LIT2137OntheP"/>
+<note>Indication for reading: <foreign xml:lang="gez">በዓለ፡ ፶፡ ምንባብ፡</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.19">
+<locus from="127v" to="131v"/>
+<title type="complete" ref="LIT6047On12ApostlesRetuaH"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.20">
+<locus from="132r" to="137v"/>
+<title type="complete" ref="LIT2145OntheT"/>
+<note>Indication for reading: <foreign xml:lang="gez">አመ፡ ፲ወ፫፡ ለነሐሲ፡ ምንባብ፡</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.21">
+<locus from="138r" to="154v"/>
+<title type="complete" ref="LIT1640Homily"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.22">
+<locus from="154v" to="163r"/>
+<title type="complete" ref="LIT2124OntheF"/>
+<note>Indication for reading: <foreign xml:lang="gez">አመ፡ ፰፡ ለኅዳር፡ ምንባብ፡</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.23">
+<locus from="163r" to="166v"/>
+<title type="incomplete" ref="LIT3959Homily"/>
+<note>Indication for reading: <foreign xml:lang="gez">አመ፡ ፲ወ፪፡ ለወርኅ፡ ኅዳር። ሚካኤል፡ ምንባብ።</foreign></note>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+
+</msContents>
+
                <physDesc>
                   <objectDesc form="Codex">
                      <supportDesc>
                         <support>
                            <material key="parchment"/>
+                           <p>Parchment repairs on <locus target=" #33 #73 #143"></locus>.</p>
                         </support>
+  <extent>
+  <measure unit="leaf">166</measure>
+  <dimensions type="outer" unit="mm">
+    <height>360</height>
+    <width>240</width>
+  </dimensions>
+  </extent>
                         <foliation/>
-                        <condition key="good"/>
+                        <condition key="deficient">The edges of the folios at the beginning and at the end are damaged.</condition>
```

Comment:
This sounds rather too insignificant for "deficient" (considering the definition) - but you looked at it and can estimate it better

Reply:
After checking the guidelines, "good" seems appropriate, the loss of text is very insignificant (the rest we cannot judge).